site/src/pages/TemplateSettingsPage/TemplateSchedulePage/ScheduleDialog.tsx

```diff
@@ -68,11 +67,12 @@ export const ScheduleDialog: FC<ScheduleDialogProps> = ({
             <>
               <h4>Dormancy Threshold</h4>
               <p css={styles.dialogDescription}>
-                This change will result in {inactiveWorkspacesToGoDormant}{" "}
-                workspaces being immediately transitioned to the dormant state
-                and {inactiveWorkspacesToGoDormantInWeek} over the next seven
-                days. To prevent this, do you want to reset the inactivity
-                period for all template workspaces?
+                This change will result in{" "}
+                <strong>{inactiveWorkspacesToGoDormant}</strong> workspace(s)
```

Comment:
```suggestion
                <strong>{inactiveWorkspacesToGoDormant}</strong> {inactiveWorkspacesToGoDormant === 1 ? "workspace" : "workspaces"}
```

we tend to avoid the parenthesized (s) in our language